chirp/drivers/tdh8.py

```diff
@@ -638,8 +937,8 @@ class TDH8(chirp_common.CloneModeRadio):
     ident_mode = b'P31183\xff\xff'
     BAUD_RATE = 38400
     NEEDS_COMPAT_SERIAL = False
-    _memsize = 0x1eef
-    _ranges_main = [(0x0000, 0x1eef)]
+    _memsize = 0x1fef
+    _ranges_main = [(0x0000, 0x1fef)]
```

Comment:
This changes the H8's memsize, which you should  not do. This probably breaks the H8 - why are you making this change?

Reply:
It will not damage H8. I have updated the test file and it was successful. This is the new function added later.

Reply:
It changes the communication with the radio. You cannot know that it works by running tests on an image. Also, you may not modify the image for the H8, only the code, and only in ways that do not alter its behavior. This changes its behavior and thus is not allowed. Set `_ranges_main` on your H3 driver if you want it to be different there.

Reply:
Thanks for the heads up, we're the vendor of H8 and H3, and since we're adding new features, the address will change, and we'll keep testing it, and if there's a problem, we'll fix it. Do you have any good suggestions?